src/transaction/flashback.h

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+
+/*
+ * flashback.h
+ *
+ */
+
+#ifndef _FLASHBACK_H_
+#define _FLASHBACK_H_
+
+#ident "$Id$"
+
+#include <map>
+
+#include "config.h"
+#include "error_manager.h"
+#include "file_io.h"
+#include "log_lsa.hpp"
+#include "thread_compat.hpp"
+
+#define FLASHBACK_MAX_SUMMARY   INT_MAX
+#define FLASHBACK_MAX_TABLE     32
+
+#define FLASHBACK_CHECK_AND_GET_SUMMARY(summary_list, trid, summary_entry) \
+  do \
+    { \
+      if ((summary_list).count(trid) != 0) \
```

Comment:
it will be very nice to use `contains`, but it only support after c++20 version. 
`find` iterates to the end of the map if element is not found, I think it costs more to use `find` than to use `count`
So, I used `count` and it works equal to `find`.